app/jobs/invoices/finalize_job.rb

```diff
@@ -5,7 +5,7 @@ class FinalizeJob < ApplicationJob
     queue_as 'invoices'
 
     def perform(invoice)
```

Comment:
The Job itself is not renamed (yet) because renaming jobs is not so easy with self-hosted users. We believe it's not worth the effort.